openapi/components/schemas/Scheduling/PeriodAnchor.yaml

```
@@ -0,0 +1,10 @@
+type: object
+description: Instruction for calculating period anchor.
```

nitpick: I think adding 'the' makes this easier to read
```suggestion
description: Instruction for calculating the period anchor.
```